src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -545,6 +545,9 @@ private function getArgumentsAsPhp(\DOMElement $node, $name, $file, $lowercase =
                     }
                     break;
                 case 'tagged':
+                    @trigger_error(sprintf('Type "%s" of Tag <%s> in "%s" is deprecated since version 4.4 and will be removed in 5.0. Use "tagged_iterator" instead.', $arg->getAttribute('type'), $name, $file), E_USER_DEPRECATED);
```

Comment:
```suggestion
                    @trigger_error(sprintf('Type "%s" of Tag <%s> in "%s" is deprecated since Symfony 4.4 and will be removed in 5.0. Use "tagged_iterator" instead.', $arg->getAttribute('type'), $name, $file), E_USER_DEPRECATED);
```